### ISSUE TITLE
”Test your skills: Other controls” article: "Finished example" not displaying

### ISSUE THREAD

**Issue by LozzyChops** [mdn/content#23776]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Other_controls

### What specific section or headline is this issue about?

any and all "Test your skills" pages

### What information was incorrect, unhelpful, or incomplete?

"Try updating the live code below to recreate the finished example." The finished example is not appearing in any of the pages that I have visited. The box I believe it is meant to appear in is empty.

### What did you expect to see?

"the finished example"

### Do you have any supporting links, references, or citations?

NA

### Do you have anything more you want to share?

NA

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/test_your_skills_colon__other_controls`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Other_controls
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/test_your_skills_colon__other_controls/index.md
* Last commit: https://github.com/mdn/content/commit/4f6ba2f5a1615fe155292cac416c7ab6b9d711ec
* Document last modified: 2022-09-11T17:33:19.000Z

</details>

**Comment by wbamberg** [comment]
Thanks for filing, @LozzyChops !

You are supposed to create the "Finished example" from what is given in the editor. For example, in the first exercise in the page you've given, you are supposed to add some HTML where it says `<!-- add your input here  -->` to complete the example:

<img width="712" alt="Screen Shot 2023-01-26 at 9 38 55 AM" src="https://user-images.githubusercontent.com/432915/214909235-0062ccd1-dc47-4c05-abcb-18d868342c60.png">

The empty box is I believe for CSS, and it is confusing, I agree.

**Comment by LozzyChops** [comment]
Ok, thank you.  

I would suggest rewording the phrase "Try updating the live code below to recreate the finished example:" because it gives the impression a finished example is present on the page and that the student should be trying to "recreate" it. Not realizing that the central box was for CSS (a label or something on that box might help noobs like me), I assumed a finished example for me to imitate should be appearing there and that something buggy was going on because it wasn't. I think maybe the most misleading parts the combination of the word "recreate" and the use of the word "the", which seems to imply there is a finished example already somewhere. Anyways, thanks again.

**Comment by wbamberg** [comment]
Yeah, it is a fair criticism and people are constantly confused by these exercises. I'll try to improve the wording here.

> Not realizing that the central box was for CSS (a label or something on that box might help noobs like me)

I actually didn't know it was for CSS - I guess and added some CSS, and it worked :). So it's non-obvious to everyone. I think we should remove the box in this case.
